disprel_split.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
+
+# Usage: ./disprel.py path/to/folder
+
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+from matplotlib import cm
+import numpy as np
+import sys
+import os.path
+from os.path import join as pjoin
+from scipy.constants import value as constants
+from glob import glob
+import re
+from tqdm import tqdm
+from aux import *
+import argparse
+
+
+parser = argparse.ArgumentParser(description='Plasma Dispersion Processor')
+parser.add_argument('--i', default=None, type=str, help='Path to E-field data')
+parser.add_argument('--periodic', default=True, type=bool, help='Period System in Y')
+parser.add_argument('--yLoc', default=4, type=int, help='Choose Y location in bounded system')
+parser.add_argument('--sN', default=4, type=int, help='No. of chuncks you want to split the processed file')
+parser.add_argument('--plot', default=True, type=bool, help='Plot the figure')
+
+args		= parser.parse_args()
+folder		= args.i
+periodic	= args.periodic
+yLoc		= args.yLoc	#8 # Choose y location
+splitNos	= args.sN #No. of chunks
```

Comment:
You may want to know that in Python (PEP8) the naming convention is usually `variable_name`, `function_name`, `ClassName`. I don't always adhere to this myself, because to be honest, I think the convention is a little bit ugly.

Reply:
Personally, I like camelCase for naming variables. If this is not absolutely necessary I would like to keep it the same.

Reply:
Okay!